src/test/rpc/LedgerData_test.cpp

```diff
@@ -264,6 +264,38 @@ class LedgerData_test : public beast::unit_test::suite
         }
     }
 
+    void testLedgerType()
+    {
+        using namespace test::jtx;
+        Env env { *this, envconfig(no_admin) };
+        Account const gw { "gateway" };
+        auto const USD = gw["USD"];
+        env.fund(XRP(100000), gw);
+
+        int const num_accounts = 10;
+
+        for (auto i = 0; i < num_accounts; i++)
+        {
+            Account const bob { std::string("bob") + std::to_string(i) };
+            env.fund(XRP(1000), bob);
+        }
+        env.close();
+
+        // no limit specified
```

Comment:
Excellent.  I'll make the same change in the 3 identical places earlier in this file.